app/common/auth/authentication-events.run.js

```diff
@@ -42,17 +42,13 @@ function AuthenticationEvents($rootScope, $location, Authentication, Session, _,
             });
     }
 
-    function doLogout(redirect) {
+    function doLogout() {
         $rootScope.currentUser = null;
         $rootScope.loggedin = false;
         // we don' wat to reload until after filters are correctly set with the backend default that the user
         // would get when logged out
         PostFilters.resetDefaults().then(function () {
-            if (redirect) {
-                $location.url(redirect);
-            }
-
-            $state.go($state.$current.name ? $state.$current : 'map', null, { reload: true });
+            $state.go('posts.map');
```

Comment:
I don't think this should be the case. If I'm on a post detail view and I log out... I should stay on the same view.

Reply:
ok. 
This was reported as a requirement/bug so I thought it was the expected behavior. Will change it..

Reply:
Hrm. I might be wrong? Where was it logged/who was it logged by?

Reply:
https://ushahidi.slack.com/archives/C03UU507R/p1510670345000387 Ah, ok I think Carolyn reported it to me in chat, and asked here. 
But we might be wrong?